SUPPORTED_PLATFORMS.md

```diff
@@ -0,0 +1,90 @@
+# Supported Platforms
+
+This document outlines the platform support matrix for different versions of the Microsoft build of Go.
+
+## Support Policy
+
+The Microsoft build of Go follows the upstream Go [Release Policy](https://go.dev/doc/devel/release#policy). This means we support each major release (1.X) until there are two newer major releases. A new Go major version is [released every six months](https://github.com/golang/go/wiki/Go-Release-Cycle), so each Go major version is supported for about one year.
+
+## Currently Supported Versions
+
+As of July 2025, the following Go versions are actively supported:
+
+- **Go 1.25** (upcoming release)
+- **Go 1.24** (current stable)
+- **Go 1.23** (previous stable)
+
+After the release of Go 1.25, Go 1.23 will no longer be supported.
+
+## Platform Support Matrix
+
+### Go 1.25 (Upcoming)
+
+| Platform | Architecture | Status | Notes |
+|----------|-------------|--------|-------|
+| Linux | amd64 | ✅ Supported | Primary platform, Azure Linux 3.0+, Ubuntu 22.04+ |
+| Linux | arm64 | ✅ Supported | Azure Linux 3.0+, Ubuntu 22.04+ |
+| Linux | armv6l | ✅ Supported | Modern Linux distributions |
```

Comment:
Unclear what Modern Linux distributions actually exist on armv6l, and would we actually support them? (All of them?)

Reply:
This one I'm not sure about, what was the original customer request for `armv6l`? Do we know what platform they required?

Reply:
Forwarded you the original email thread.